src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -167,8 +167,6 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
                 if ($inAttributes = isset($normalizationContext[AbstractNormalizer::ATTRIBUTES][$association])) {
                     // prepare the child context
                     $normalizationContext[AbstractNormalizer::ATTRIBUTES] = $normalizationContext[AbstractNormalizer::ATTRIBUTES][$association];
-                } else {
-                    unset($normalizationContext[AbstractNormalizer::ATTRIBUTES]);
```

Comment:
@dunglas I think you added this but it does not look useful to me, I'd be :+1: for this patch, any reason we should keep this?